### PR TITLE
remove styleguide files

### PR DESCRIPTION
**Pull request by peter-mouland**
**Why?**
 - having things like fonts in style-guide and react-toolkit means some apps are incorrect
 - having duplicate components is confusing 
 - Having react components for things that are just HTML adds to the complexity for no gain
 - Keeping constant with react will make using components easier in the future 
   - i.e. use className prop instead of the custom classes
   - use onClick prop of custom handleChange functions

**What have we done!**
- always pass non-component specific props straight into the html
- remove files that are causing conflicts with the style guide 
  - [x] font styles
  - [x] docs styles
  - [x] SVG Icons
  - [x] font files

- Removed CSS only components
  - [x] Buttons
  - [x] Checkboxes
  - [x] Radio Buttons
  - [x] inputs
  - [x] Dropdowns

- Kept components that have nested HTML or have event handling
   - Accordion
   - Header
   - Notification
   - Spinner
   - Ticket
   - Tip
   - Toggle Button

- Added an `Svg` component which displays SVG's inline
   - This allows us to use CSS to change styles of icons
   - This works server-side and client-side

- Removed `classes` prop in favour of `className`
  - this is another major change, but keeps our components consistent with React

- Ensure ALL components propagate ALL unknown properties
  - i.e. passing in an unknown property (like 'id' or 'data-whatever') will always be applied to the outer div of the component

- Combined webpack configs into a single file (as it was getting confusing)

### CODE REVIEW

**Comment by bentidy** [comment]
Are we going to remove the font files too? Seeing as they don't seem to be used elsewhere.

**Comment by nowarkdev** [comment]
@alexflorisca @benstokoe You might as well switch all of the sass from toolkit to styleguide and bump the major version number instead of loads of breaking changes.

**Comment by nowarkdev** [comment]
@ariley90 @malcs-sky We agree with the above, there shouldn't be a need for any styles in this project now that we are using the style guide :+1:

**Comment by peter-mouland** [comment]
we've done a PR to help with this..
https://github.com/sky-uk/nowtv-styleguide/pull/50

File: docs/routes.js [hunk 1]
```diff
@@ -21,20 +21,40 @@ class Bugs extends React.Component {
...
-    },
-    { path: '/bugs',
-      component: Bugs
-    },
-    { path: '/bugs/accordionWithButtons',
-      component: AccordionWithButtons
-    }
-  ]
-};
+//export default {
```
Comment (nowarkdev) [inline docs/routes.js]: Delete this

Comment (alexflorisca) [inline docs/routes.js]: Delete this if not needed

**Comment by alexflorisca** [comment]
Mahoosive PR, nice work. Only thing I noticed is that the dropdown seems to have vanished (not in the styleguide either)

**Comment by NowTV-Assurance** [comment]
this styleguide? https://nowtv-styleguide.herokuapp.com/

**Comment by alexflorisca** [comment]
Ok. I think there's LOADS of breaking changes so upgrading might be a time consuming process - think it's worth it in the long run though so if teams could put half a day - a day in their back logs that would get us all on the same page. :+1:

**Comment by peter-mouland** [comment]
@ldabiralai @nwc1991 @ariley90 @malcs-sky you guys able to review?

**Comment by bentidy** [comment]
:+1: 

@ldabiralai @bdo

**Pull request merged**